WrightTools/data/_andor.py

```diff
@@ -0,0 +1,109 @@
+"""Andor."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+import os
+import time
+
+import numpy as np
+
+from ._data import Axis, Channel, Data
+from .. import exceptions as wt_exceptions
+from ..kit import _timestamp as timestamp
+
+
+# --- define --------------------------------------------------------------------------------------
+
+
+__all__ = ["from_andor"]
+
+
+# --- from function -------------------------------------------------------------------------------
+
+
+def from_andor(filepath, name=None, parent=None, verbose=True):
+    """Create a data object from Andor CCD spectrometer.
+
+    Parameters
+    ----------
+    filepath : string, list of strings, or array of strings
+        Path to .txt file.
+    name : string (optional)
+        Name to give to the created data object. If None, filename is used.
+        Default is None.
+    parent : WrightTools.Collection (optional)
+        Collection to place new data object within. Default is None.
+    verbose : boolean (optional)
+        Toggle talkback. Default is True.
+
+    Returns
+    -------
+    data
+        New data object(s).
+    """
+    # parse filepath
+    if not filepath.endswith("asc"):
+        wt_exceptions.WrongFileTypeWarning.warn(filepath, "asc")
+    # parse name
+    if not name:
+        name = os.path.basename(filepath).split(".")[0]
+    # create data
+    with open(filepath) as f:
+        axis0 = []
+        arr = []
+        attrs = {}
+        while True:
+            line = f.readline().strip()[:-1]
+            if len(line) == 0:
+                break
+            else:
+                line = line.split(',')
+                line = [eval(x) for x in line]
```

Comment:
`eval` considered harmful...

do we need `eval`? I've just got to imagine that there is a better way to parse this info

at the very least, can we use [`ast.literal_eval`](https://docs.python.org/2/library/ast.html#ast.literal_eval) instead?